test/parallel/test-stream-finished.js

```diff
@@ -312,7 +312,6 @@ testClosed((opts) => new Writable({ write() {}, ...opts }));
   }));
 }
 
-
```

Comment:
Unrelated whitespace change?

Reply:
Yea, there was an extra whitespace that shouldn't be there...